caddyhttp/httpserver/plugin.go

```diff
@@ -453,6 +453,7 @@ var directives = []string{
 	// directives that add middleware to the stack
 	"locale", // github.com/simia-tech/caddy-locale
 	"log",
+	"nobots",    // github.com/Xumeiquer/nobots
```

Comment:
You'll probably want to be after `rewrite` though, since rewrite could change the URI that your middleware reads. So consider putting this after `status` or `cors` instead. Also be sure to use `go fmt` so the CI tests pass!